### DISCUSSION TITLE
Passing Dynamic Variable Into Send Request URL

### DISCUSSION THREAD

**Discussion by usgrant4** [postmanlabs/postman-flows#356] [Q&A] [answered]
Do I have the ability to pass in a dynamic variable from the response body, **reportId**, to a send request url that requires the **reportId**? I have attempted to pass the reportId directly and with a durable with a form of different syntaxes but it does not seem to translate. I have tried {{reportId}}, {{body.reportId}}. etc. 

EX: https://api.partnercenter.microsoft.com/insights/v1/mpn/ScheduledReport/execution/{{$reportId}}

![image](https://user-images.githubusercontent.com/48367714/181605183-30ea14eb-cbe3-4024-990b-4ff415d8242d.png)

![image](https://user-images.githubusercontent.com/48367714/181605304-3513536b-b8eb-4801-8c29-ba51cef810d1.png)

![image](https://user-images.githubusercontent.com/48367714/181605831-f21a049d-13a9-41cb-85ca-3cc1be7985f4.png)

**Comment by samay-v** [comment, marked as answer]
You should be able to send data to a request like this. Here's an example to help you out https://www.postman.com/postman/workspace/postman-answers/flow/62e37bc54ef8800b3a6547cd

**Reply by usgrant4** [reply]
Thanks, so you can't pass in a durable to a url but you can directly pass in a var directly. Adding a query param passed in like below so I removed that and I'm good to go. 

![image](https://user-images.githubusercontent.com/48367714/181790405-b702d85a-810b-4ba5-a9b0-0c15046698d0.png)


![image](https://user-images.githubusercontent.com/48367714/181790050-893aa4fc-0da5-43bf-8145-cbd4bb31655f.png)
